source/libraries/HelperFunctions.ts

```diff
@@ -1,9 +1,12 @@
 import * as binascii from "binascii";
 import * as EthjsAccount from "ethjs-account";
+import * as EthjsQuery from 'ethjs-query';
 
-const DEFAULT_TEST_ACCOUNT_BALANCE = 1 * 10 ** 20; // Denominated in wei
+const DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS: number = 1200;
+const ETHEREUM_BLOCK_INTERVAL_MILLISECONDS: number = process.env.ETHEREUM_BLOCK_INTERVAL_MILLSECONDS ? parseInt(process.env.ETHEREUM_BLOCK_INTERVAL_MILLISECONDS!, 10) : DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS;
```

Comment:
Recommend renaming `ETHEREUM_BLOCK_INTERVAL_MILLISECONDS` to `ETHEREUM_POLLING_INTERVAL_MILLISECONDS`.  This variable isn't used as the block interval, it is used as the polling interval.

Reply:
Also recommend inlining `DEFAULT_ETHEREUM_BLOCK_INTERVAL_MILLISECONDS` since it is only used once and its usage context makes it obvious that its a default.